src/tasks/DockerTaskProvider.ts

```diff
@@ -51,7 +50,6 @@ export abstract class DockerTaskProvider implements TaskProvider {
                 context.platform = getPlatform(task.definition);
 
                 context.actionContext.telemetry.properties.dockerPlatform = context.platform;
-                context.actionContext.telemetry.properties.orchestration = 'single' as DockerOrchestration; // TODO: docker-compose, when support is added

```

Comment:
No longer needed, as the task name is an exact hint as to whether we're dealing with single or compose orchestration.